docker/spark/Dockerfile

```diff
@@ -29,4 +29,5 @@ ENV SPARK_HOME /spark
 ENV PATH="/spark/bin:${PATH}"
 
 ADD scripts /scripts
+RUN chmod 777 /scripts
```

Comment:
chmod +x /scripts

Reply:
Actually what are you trying to achieve?

Reply:
I don't need it anymore. 
(at first, I was trying to overwrite the original "finish-submit" script)